index.html

```diff
@@ -811,25 +810,20 @@ <h3>The <dfn>toJSON</dfn> member</h3>
   <h3>The <dfn>PressureObserverOptions</dfn> dictionary</h3>
   <pre class="idl">
     dictionary PressureObserverOptions {
-      double sampleRate = 1.0;
+      [EnforceRange] unsigned long long sampleInterval = 0;
     };
   </pre>
   <section>
-    <h3>The <dfn>sampleRate</dfn> member</h3>
+    <h3>The <dfn>sampleInterval</dfn> member</h3>
     <p>
-      The {{PressureObserverOptions/sampleRate}} member represents the <dfn>requested sampling
-      rate</dfn> expressed in Hz, ie. it represents the number of samples requested to be obtained
-      from the hardware per second. The [=reporting rate=] will never exceed the [=requested sampling rate=].
+      The {{PressureObserverOptions/sampleInterval}} member represents the <dfn>requested sample
+      interval</dfn> expressed in milliseconds, ie. it represents the requested interval between samples
+      to be obtained from the hardware. The [=reporting rate=] will never exceed the sampling rate which
+      is obtained by dividing 1000 by the [=requested sample interval=].
```

Comment:
" The [=reporting rate=] will never exceed the sampling rate which is obtained by dividing 1000 by the [=requested sample interval=]."

I think there is a wrong order of words in the "by dividing ..."
I m still not understanding the meaning of the sentence.

Reply:
but you "divide by" something: https://dictionary.cambridge.org/dictionary/english/divide-by

Reply:
Let me ask a native english speaker

Reply:
Native speaker said "looks good to me, like literally spot on" - so I guess we are fine :)